Source/WebCore/editing/Editor.cpp

```diff
@@ -2279,6 +2285,16 @@ void Editor::setComposition(const String& text, const Vector<CompositionUnderlin
         }
     }
 
+    if (auto* cache = m_document.existingAXObjectCache()) {
+        auto* currentCompositionNode = compositionNode();
+        // Clear the previous cache, if it's a different node
+        if (previousCompositionNode && previousCompositionNode != currentCompositionNode)
+            cache->onTextCompositionChange(*previousCompositionNode);
+        // Then update the new cache with current information.
+        if (currentCompositionNode)
+            cache->onTextCompositionChange(*currentCompositionNode);
```

Comment:
"Clear the previous cache" and "update the new cache" read to me like we're modifying the AXObjectCache, but really we're modifying the compositions for the previous and current nodes. IMO, you could probably just remove these comments — I think your descriptive variable names and function name describe what is happening well enough.

Reply:
Yeah, just gonna remove them. would like to keep my Editor.cpp footprint low.